main.go

```diff
@@ -154,6 +154,64 @@ func validatePeacemakrCiphertext(sdk peacemakr_go_sdk.PeacemakrSDK, from *os.Fil
 	}
 }
 
+func signOrFail(sdk peacemakr_go_sdk.PeacemakrSDK, from, to *os.File) {
+	if from == nil {
+		log.Fatalf("missing 'from' in signing")
+	}
+
+	if to == nil {
+		log.Fatalf("missing 'to' in signing")
+	}
+
+	if from == to {
+		log.Fatalf("in-place decryptiosigningn is not supproted (from and to are the same)")
```

Comment:
Love the actionable error messages here, just friendly heads up,

in-place what is not supported? :-p

Reply:
hahah good catch :-p